### ISSUE TITLE
Running in Virtualenv

### ISSUE THREAD

**Issue by svartkanin** [jason-kane/PyYapf#43]
When `yapf` is installed in a virtualenv only then it's quite cumbersome to change the execution path all the time. Would be great if there's a way of automatically picking `yapf` installed in the current virutalenv of the file to be formatted.

**Comment by catch22** [comment]
If you run sublime from the virtualenv it should work (but that's cumbersome, too). Would you know of a better way of picking up the virtualenv for a given file?

You can also override the path to yapf on a per-project basis.

**Comment by svartkanin** [comment]
I've tried putting the custom command as execution (which doesn't require to be in a virtualenv):

```
{
    "yapf_command": "pipenv run yapf"
}
```

But that gives me the error

```
OSError: [Error 2] No such file or directory: 'pipenv run yapf'
```

**Comment by jason-kane** [comment]
I assume we aren’t doing the popen in a way that is friendly to commands
with arguments.  The stupid workaround is a executable shell script that
runs “pipenv run yapf $1” and put the script in the yapf_command.

Or fix it yourself, probably won’t be hard.  I suspect splitting the
command so the first list item is “pipenv” is the key.  Or wait a day or
two and I’ll fix it.  Or maybe Michael beats me to it.

**Comment by svartkanin** [comment]
Great I'll try it out as soon as it's released!

**Comment by jason-kane** [comment]
Okay, should be addressed in v0.14.1.  Please post again if it does work (or to confirm it does).  It will be a few hours before the new code shows up in sublime Package Control.
